python/pyspark/sql/tests/connect/test_parity_arrow.py

```diff
@@ -31,8 +31,17 @@ def test_createDataFrame_fallback_disabled(self):
     def test_createDataFrame_fallback_enabled(self):
         super().test_createDataFrame_fallback_enabled()
 
-    def test_createDataFrame_with_map_type(self):
-        self.check_createDataFrame_with_map_type(True)
+    def test_createDataFrame_pandas_with_map_type(self):
+        self.check_createDataFrame_pandas_with_map_type(True)
+
+    def test_createDataFrame_pandas_with_struct_type(self):
+        self.check_createDataFrame_pandas_with_struct_type(True)
+
+    def test_createDataFrame_arrow_with_struct_type(self):
```

Comment:
We can remove this if that takes no argument. It will be inherited, and run the tests

Reply:
Ah, right. Done in 292d3c8. Thanks.